test/forms/inputs/date-input.test.js

```diff
@@ -38,9 +40,14 @@ test('DateInput updates the value on change', async () => {
   render(<WrappedDateInput />)
 
   const input = screen.getByRole('textbox', { name: 'Field' })
-  await user.type(input, '02/02/2023{Enter}')
 
-  expect(input).toHaveValue('02/02/2023')
+  await user.click(input)
```

Comment:
I don't think you need `user.click` here, as `user.type` already focuses on the element. The test passes without it.

Reply:
Does it open the popover menu though? I _thought_ in doing testing earlier that it didn't open, but honestly I was trying a whole lot of things so maybe I was mistaken. I remember that my intent was to try and mimic the user's experience as much as possible